### ISSUE TITLE
[BUG] Lawnchair 14 Beta 2 does not contain the latest translations

### ISSUE THREAD

**Issue by HarmfulBreeze** [Deletescape-Media/Lawnchair#4210]
### Describe the bug

Hi!

It looks like the latest release of Lawnchair does not include the latest translations from Crowdin. What's especially odd is that the commit in question has actually been merged into 14-beta, and the associated GitHub Actions artifact does contain the latest translations: https://github.com/LawnchairLauncher/lawnchair/commit/cc0ecd370e0694b5b8da27d1ad14088be01c78fd

I'm really glad I was able to contribute to Lawnchair with professional-grade French translations. Let's put them into the app :)

### Steps to reproduce

N/A

### Expected behavior

Translations are up-to-date.

### Device information

- App version: 14 Beta 2

**Comment by MrSluffy** [comment]
This is not a bug

**Comment by HarmfulBreeze** [comment]
@MrSluffy Should I open a discussion on GitHub or something? I can't tell if this is supposed to be WAD or just that I shouldn't report this here.

**Comment by HarmfulBreeze** [comment]
@MrSluffy I would appreciate an answer on this. Treating people who spend time on your project as such is pretty uncool and I might remove my contributions otherwise.

**Comment by SuperDragonXD** [comment]
Regarding this, could you give an example preference that isn't updated to the latest translation? It seems odd that this can happen.

**Comment by HarmfulBreeze** [comment]
Hi! Thank you for looking into my case :)

In the following screenshot from Lawnchair 14 Beta 2, the third option in the popup menu states "Paramètres de l'écran d'accueil".

![Screenshot_20240407-123504](https://github.com/LawnchairLauncher/lawnchair/assets/6281092/cfb1f3c2-92fb-4a9e-88e2-45899944b62b)

This string has been updated to "Paramètres de l'accueil" on Crowdin. This new translation matches the one in Pixel Launcher and actually fits the bubble.
The build from GitHub Actions artifacts that I mentioned in the opening post does contain this updated translation (and others).

Therefore I believe something may be wrong with APKs from GitHub releases.

**Comment by SuperDragonXD** [comment]
Yeah looks like there are some issues with translations in that release. 

Unfortunately, I can't really look into this issue properly unless we make a new release and do additional tests. I'll try to test making a release locally soon and see if the text changes, but I'll re-open the issue as of now.

You may temporarily use the development builds (or Goooler’s nightly releases compiled as `release`) to have this translation.

**Comment by HarmfulBreeze** [comment]
Thank you for acknowledging the issue, I appreciate it. I'm happy to know that it will be worked on.

**Comment by HarmfulBreeze** [comment]
@SuperDragonXD I was able to track down the bad commit: https://github.com/LawnchairLauncher/lawnchair/commit/067fd11a09456e78952e231886324f15bd584c90

**Comment by SuperDragonXD** [comment]
Nice catch! Seems like I went a little overboard with merging some conflicts, and accidentally reverted some changes over there.

This will be alleviated with the next release (can't give ETAs yet). (If you can't wait, you may use the nightly builds to have the latest translation.)

**Comment by HarmfulBreeze** [comment]
I'm all good, thanks. As long as the issue is fixed with the next release, I'm fine with it.
Thank you for your positive attitude :)

**Comment by SuperDragonXD** [comment]
Hello. I think this issue should now be resolved in our later release. As such, I'm closing this issue.
